### PR TITLE
Tweak design

### PR DESCRIPTION
**Pull request by hosmelq**
This is the new look.

screen shot 2017-10-03 at 1 55 40 am

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -22,14 +22,92 @@ h1 {
...
-  fill: #2cbe4e;
+.pr .repo {
+  color: #0366d6;
 }
 
-.pr > .closed > svg {
-  fill: #6f42c1;
+.pr .title {
+  color: #959da5;
 }
```
Comment (jenkoian) [inline public/css/main.css]: I want to favour utility classes over css if possible, I think most of what you've done here can be done with utility classes. Take a look at http://tachyons.io/docs/

There may be some instances where you need to drop into CSS but would like to try and avoid it if possible.

**Comment by jenkoian** [comment]
Hey @hosmelq thanks for taking the time to make these changes, few bits of feedback from me:

* I like the button sitting right up next to the input, but can we keep the orange colour rather than red?

* Can we keep the colour of the PR icons? Green for open, purple for merged, red for closed?

* Can we remove the spacing from between the result rows?

**Comment by hosmelq** [comment]
Sure, I’ll make the changes.

**Comment by hosmelq** [comment]
Done.

screen shot 2017-10-03 at 1 03 21 pm

**Comment by jenkoian** [comment]
Pushed a few tiny tweaks in #151 but awesome work, thanks @hosmelq !

**Comment by jenkoian** [comment]
@hosmelq I'm so sorry, I seem to have messed up my PR by squashing the commits thus meaning your contributions are not attributed to you. Just trying to think how to fix this. How about you do a PR reverting #151, then do another PR straight after making the changes again? Sorry, know it's a faff, apologies for messing it up, but think you deserve the proper credit 👍

**Comment by hosmelq** [comment]
Do not worry for that, it's ok 🙂

**Pull request closed without merging**